app/src/main/java/com/jerboa/api/Http.kt

```diff
@@ -24,6 +24,8 @@ const val DEFAULT_INSTANCE = "lemmy.ml"
 const val MINIMUM_API_VERSION: String = "0.18"
 val REDACTED_QUERY_PARAMS = setOf("auth")
 val REDACTED_BODY_FIELDS = setOf("jwt", "password")
+val TEMP_RECOGNISED_AS_LEMMY_INSTANCES = mutableSetOf<String>()
+val TEMP_NOT_RECOGNISED_AS_LEMMY_INSTANCES = mutableSetOf<String>()
```

Comment:
Where are these getting filled? Should you just use the hardcoded `DEFAULT_INSTANCES` ?

Globals should probably go inside the singleton if you really need them, but I don't see why you do, because `DEFAULT_INSTANCES` already exists.

Reply:
I moved it and checkIfLemmyInstance to the singleton